apps/backend/src/app/services/mail/mail.ts

```diff
@@ -27,8 +27,19 @@ export async function sendMail(options: MailOptions) {
 
         return await transporter.sendMail(mailOptions);
 
-
     } catch (err) {
         return err
     }
-};
\ No newline at end of file
+};
+
+export async function sendEmailValidacion(email: string, nombre: string, validationToken: string) {
+    //Se realiza el envio del mail de verificación de la cuenta ;
+    const url = `http://localhost:4200/login/activacion-cuenta/${validationToken}`;
```

Comment:
parametrizaría el host.
```suggestion
    const url = `${env.APP_HOST}/login/activacion-cuenta/${validationToken}`;
```

Reply:
se realiza sugerencia con los datos de environment